README.md

```diff
@@ -85,7 +85,7 @@ Example configuration parameters could be:
 ```
 use_singularity          => true,
 force_singularity_jobs   => true,
-singularity_image_expr   => "/images/myimage.img",
+singularity_image_expr   => '"/images/myimage.img"',
```

Comment:
Many quotes in here, would be better to leave as-is and instead add quotes in the template, would it not?

Reply:
No, since I may want to write there:
```
singularity_image_expr => 'ifThenElse(TARGET.ContainerOS is "Ubuntu1604", "/somewhere/Ubuntu1604", "/somewhere/SL6")',
```
`singularity_image_expr` is an HTCondor expression, and not supposed to be used as string.

Reply:
See e.g. an example here:
http://research.cs.wisc.edu/htcondor/manual/v8.7/SingularitySupport.html#x45-3920003.17
on how to use it. Enforcing double quotes in the template breaks the official examples.